### DISCUSSION TITLE
Μέρος Γ - ΓΕ1

### DISCUSSION THREAD

**Discussion by std159312** [sdy60/syllabus#94] [Q&A] [answered]
Καλησπέρα σας, στο μέρος Γ εκτός από τα  screenshots από την εκτέλεση θα χρειαστεί να ανεβάσουμε και το html αρχείο που κάνει export το bitmelo?
Ευχαριστώ

**Comment by epidrome** [comment, marked as answer]
Εκτός από τις ενδεικτικές οθόνες μέσα στην αναφορά μπορείτε να δώσετε και ένα δημόσιο σύνδεσμο στο εκτελέσιμο ώστε να έχουν όλοι πρόσβαση

**Reply by std159312** [reply]
Ευχαριστώ
